crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -290,6 +296,7 @@ impl TxState {
             .unwrap_or(RowState::Absent)
     }
 
+    #[tracing::instrument(skip_all)]
     pub fn get_row(&self, table_id: &TableId, row_id: &RowId) -> Option<&ProductValue> {
```

Comment:
In places like this we can add:

```rust
tracing::info!(table_id= table_id, ...)
```

So it shows in the `values` section of `tracy`.